.gitignore

```diff
@@ -0,0 +1,2 @@
+# .gitignore
+node_modules/
```

Comment:
- [ ] Please make sure to add the 'test.md' file in your `.gitignore` file so that git knows not to track this file in your project.